src/plugins/profiling/server/routes/search_flamechart.ts

```diff
@@ -91,216 +310,10 @@ export function registerFlameChartSearchRoute(
-              body: {
-                query: filter,
-                size: 0,
-                track_total_hits: true,
-              },
-            });
-            const sampleCountFromInitialExp = resp.body.hits.total?.value as number;
-
-            logger.info('sampleCountFromPow6 ' + sampleCountFromInitialExp);
-
-            return getSampledTraceEventsIndex(
-              targetSampleSize,
-              sampleCountFromInitialExp,
-              initialExp
-            );
-          }
-        );
-        logger.info('EventsIndex ' + eventsIndex.name);
-
-        // Using filter_path is less readable and scrollSearch seems to be buggy - it
-        // applies filter_path only to the first array of results, but not on the following arrays.
-        // The downside of `_source` is: it takes 2.5x more time on the ES side (see "took" field).
-        // The `composite` keyword skips sorting the buckets as and return results 'as is'.
-        // A max bucket size of 100000 needs a cluster level setting "search.max_buckets: 100000".
-        const resEvents = await logExecutionLatency(
-          logger,
-          'query to find DownsampledIndex',
-          async () => {
-            return await esClient.search({
-              index: eventsIndex.name,
-              size: 0,
-              query: filter,
-              aggs: {
-                group_by: {
-                  composite: {
-                    size: 100000, // This is the upper limit of entries per event index.
-                    sources: [
-                      {
-                        traceid: {
-                          terms: {
-                            field: 'StackTraceID',
-                          },
-                        },
-                      },
-                    ],
-                  },
-                  aggs: {
-                    sum_count: {
-                      sum: {
-                        field: 'Count',
-                      },
-                    },
-                  },
-                },
-                total_count: {
-                  sum: {
-                    field: 'Count',
-                  },
-                },
-              },
-            });
-          }
-        );
-
-        const totalCount: number = resEvents.body.aggregations?.total_count.value;
-        const stackTraceEvents = new Map<StackTraceID, number>();
-
-        let docCount = 0;
-        let bucketCount = 0;
-        resEvents.body.aggregations?.group_by.buckets.forEach((item: any) => {
-          const traceid: StackTraceID = item.key.traceid;
-          stackTraceEvents.set(traceid, item.sum_count.value);
-          docCount += item.doc_count;
-          bucketCount++;
-        });
-        logger.info('query time registered by ES on events ' + resEvents.body.took);
-        logger.info('events document count ' + docCount);
-        logger.info('total events count ' + totalCount);
-        logger.info('unique events ' + bucketCount);
```

Comment:
These were lost in the function re-ordering, they are now back to console.log.
@jbcrail you can get pre-commit hook that will lint those with

```
node ./scripts/register_git_hooks.js
```